nuxeo-runtime/nuxeo-runtime-kv/src/test/java/org/nuxeo/runtime/kv/AbstractKeyValueStoreTest.java

```diff
@@ -115,6 +120,47 @@ public void testPutGet() {
         assertEquals(Collections.emptySet(), storeKeys());
     }
 
+    /**
+     * @since 9.10
+     */
+    protected void assertMapDeepEquals(Map map1, Map map2) {
+        assertTrue(String.format("Map1:%s and Map2:%s are not deepEquals", map1, map2),
+                new FastHashMap(map1).equals(new FastHashMap(map2)));
```

Comment:
I don't get what `FastHashMap` brings over `HashMap`. Also, map comparison can be just done with `assertEquals(map1, map2)`